snakemake/sourcecache.py

```diff
@@ -346,7 +327,7 @@ def runtime_cache_path(self):
 
     def open(self, source_file, mode="r"):
         cache_entry = self._cache(source_file)
-        return self._open(LocalSourceFile(cache_entry), mode)
+        return self._open(cache_entry, mode)
```

Comment:
Please revert this to what we had before. The LocalSourceFile for the cache entry is intentional.

Reply:
reverted to the old version